201-vm-copy-index-loops/azuredeploy.json

```diff
@@ -75,11 +76,38 @@
         "managed": true
       }
     },
+    {
+      "comments": "Default Network Security Group for template",
+      "type": "Microsoft.Network/networkSecurityGroups",
```

Comment:
this needs to account for the windows os as well

Reply:
Done.